src/components/templates/Preplaya/Preplaya.tsx

```diff
@@ -247,6 +249,11 @@ const Preplaya = () => {
             <FontAwesomeIcon icon={faMinus} className="icon" />
           </div>
         </div>
+        <div style={{ position: "fixed", top: 200, left: 27 }}>
```

Comment:
does this left:27 have to match up with the left:27 in Camp.tsx? If so, consider setting a variable in a shared .scss file (or use constants.scss).